components/CatCard.tsx

```diff
@@ -127,56 +130,73 @@ export const CatCard = (props: CatCardProps) => {
       <View style={styles.container}>
         <View style={{ flexDirection: 'row', alignItems: 'center' }}>
           <View style={{ flexDirection: 'row', alignItems: 'baseline' }}>
-            <Text style={styles.baseText}>{props.index}.</Text>
-            <Text style={styles.title}>Sex </Text>
+            <Text style={styles.baseText}>{index}.</Text>
+            <Text style={styles.title}>Sex</Text>
           </View>
           <View style={styles.circleView}>
-            <Text style={styles.genderText}>{props.cat.sex}</Text>
+            <Text style={styles.genderText}>{cat.sex}</Text>
           </View>
         </View>
-        {props.cat.notes && (
-          <Text style={styles.notes}>Observatii: {props.cat.notes}</Text>
+        {!!cat.notes && (
```

Comment:
If `notes` is a required field, it's necessary to the `!!` operator?

Reply:
Notes might be empty, and then it leaves a lot of white space in the screen due to the padding and margin.
So I'd rather we don't render the text field at all when `notes` is empty for visual purposes.